.github/workflows/go.yml

```diff
@@ -16,10 +16,23 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.15
+        go-version: 1.16
 
     - name: Build
       run: go build -v ./...
 
     - name: Test
       run: go test -v ./...
+
+  lint: 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Lint
+        run: |
+          docker run --rm -v `pwd`:/go/src/kubernetes-sigs/depstat -w /go/src/kubernetes-sigs/depstat \
```

Comment:
just wondering why are you running this in docker and not directly on the worker?

Reply:
That's what I was asking 😅  I saw it was being done this way in [klog](https://github.com/kubernetes/klog/blob/dda20801a652310157c8d60075f5605a73b68615/.github/workflows/test.yml#L22-L33) so I thought this is how it is done. Should I follow the method in the [docs](https://github.com/golangci/golangci-lint-action#how-to-use) instead?
```
- name: Run golangci-lint
  uses: golangci/golangci-lint-action@v2.5.2
```

Reply:
interesting, I always run this on worker :D 

Looking at the git blame in klog it looks like they migrated to github actions from something else so maybe they just moved over the docker command as they were running on different platform?

Reply:
That could be the case 🤔 I guess then it is better for us to run it as the docs suggest.
cc @dims

Reply:
so now, if we do it like this, how does the linter find a configuration? (so that we can enable/disable some linters)? (I did not check the docs or rather did not see it directly in readme)

Reply:
I don't want you to keep changing this over an over but now that I am thinking about it the best approach might be to introduce simple make file and then just run the make command from the actions. That way you will be able to run the linter locally as well in the same way the CI runs it which makes debugging easier

Reply:
+1 to what @alenkacz said.

Reply:
i was pointing to klog as a way to show how GH actions are added to project and not necessarily do EXACTLY what it did :)